server/lorax_server/models/__init__.py

```diff
@@ -325,6 +328,19 @@ def get_model(
                 trust_remote_code=trust_remote_code,
             )
         raise NotImplementedError("Qwen model requires flash attention v2")
+    
+    if model_type == "phi-msft" or model_type == "phi":
```

Comment:
what do you think about using model_type in [] instead ?

Reply:
Done.